resources/public/api/conf/1.0/employment_expenses_create_and_amend.yaml

```diff
@@ -0,0 +1,111 @@
+summary: Create and Amend Employment Expenses
+description: |
+  This endpoint enables you to add or update existing employment expenses. This endpoint can only be used after the tax 
```

Comment:
According to the spec this description should be slightly different: `This endpoint allows a developer to add or update existing employment expenses. This endpoint can only be used after the tax year has ended. A National Insurance number and Tax year are required.

Amending an expense will override the expense and remove the "dateIgnored" flag from HMRC expenses.`

Reply:
(just the first sentence)